### ISSUE TITLE
mainapp.Shutdown undefined (type *appng.App has no field or method Shutdown)

### ISSUE THREAD

**Issue by UvDream** [wailsapp/wails#1594]
### Description

mainapp.Shutdown undefined (type *appng.App has no field or method Shutdown)

### To Reproduce

```
wails init -n myproject -t react-ts
```
Error when initializing the project

### Expected behaviour

can run

### System Details

```shell
Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:		MacOS
Version: 	12.4
ID:		199506
Go Version:	go1.18.4
Platform:	darwin
Architecture:	arm64

Wails
------
Version: 	v2.0.0-beta.38

Dependency			Package Name	Status		Version
----------			------------	------		-------
xcode command line tools 	N/A		Installed	2395
npm 				N/A		Installed	8.1.0
*upx 				N/A		Available
*nsis 				N/A		Available

* - Optional Dependency


mac M1 max
```

**Comment by UvDream** [comment]
I solved it, should not `go run main.go` to run, should `waits dev`,I have a suggestion, the documentation can be put on this [crowdin](https://tauri.crowdin.com/u) website,I want to participate in the translation of documents and promote this tool

**Comment by leaanthony** [comment]
This is fixed on master. Please use master until the next release (might be tonight!) 👍

**Comment by leaanthony** [comment]
> I solved it, should not `go run main.go` to run, should `waits dev`,I have a suggestion, the documentation can be put on this [crowdin](https://tauri.crowdin.com/u) website,I want to participate in the translation of documents and promote this tool

@misitebao has opened a project [here](https://crowdin.com/project/wails). Please contact him 👍

**Comment by UvDream** [comment]
> This is fixed on master. Please use master until the next release (might be tonight!) 👍

Looking forward to the official version very much, looking forward to making a little contribution to the document

**Comment by leaanthony** [comment]
I've just released `v2.0.0-beta.39`. Please update using `wails update -pre` 👍

**Comment by BigCatGit** [comment]
The problem has been solved after the update
